apps/wallet/src/ui/app/shared/tooltip/index.tsx

```diff
@@ -135,7 +135,7 @@ export function Tooltip({ tip, children, placement = 'top' }: TooltipProps) {
                             }}
                             {...getFloatingProps({ ref: floating })}
                         >
-                            <div className="flex flex-col flex-nowrap gap-px rounded-md bg-gray-100 p-2 z-50">
+                            <div className="flex flex-col flex-nowrap gap-px rounded-md bg-gray-100 p-2">
```

Comment:
the z-index here wasn't doing anything since this element has static positioning